CatNote.BLL/Exceptions/BadRequestException.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CatNote.BLL.Exceptions;
+
+public class BadRequestException : Exception
```

Comment:
Please, implement the rest of the ctors

https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions